src/NotificationTargetCommonITILObject.php

```diff
@@ -1224,19 +1227,19 @@ public function getDataForObject(CommonDBTM $item, array $options, $simple = fal
                     $suppliers[] = $supplier->getName();
 
                     $tmp = [];
-                    $tmp['##supplier.id##']       = $sid;
-                    $tmp['##supplier.name##']     = $supplier->getName();
-                    $tmp['##supplier.email##']    = $supplier->getField('email');
-                    $tmp['##supplier.phone##']    = $supplier->getField('phonenumber');
-                    $tmp['##supplier.fax##']      = $supplier->getField('fax');
-                    $tmp['##supplier.website##']  = $supplier->getField('website');
-                    $tmp['##supplier.email##']    = $supplier->getField('email');
-                    $tmp['##supplier.address##']  = $supplier->getField('address');
-                    $tmp['##supplier.postcode##'] = $supplier->getField('postcode');
-                    $tmp['##supplier.town##']     = $supplier->getField('town');
-                    $tmp['##supplier.state##']    = $supplier->getField('state');
-                    $tmp['##supplier.country##']  = $supplier->getField('country');
-                    $tmp['##supplier.comments##'] = $supplier->getField('comment');
+                    $tmp['##supplier.id##']        = $sid;
+                    $tmp['##supplier.name##']      = $supplier->getName();
+                    $tmp['##supplier.email##']     = $supplier->getField('email');
+                    $tmp['##supplier.phone##']     = $supplier->getField('phonenumber');
+                    $tmp['##supplier.fax##']       = $supplier->getField('fax');
+                    $tmp['##supplier.website##']   = $supplier->getField('website');
+                    $tmp['##supplier.email##']     = $supplier->getField('email');
+                    $tmp['##supplier.address##']   = $supplier->getField('address');
+                    $tmp['##supplier.postcode##']  = $supplier->getField('postcode');
+                    $tmp['##supplier.town##']      = $supplier->getField('town');
+                    $tmp['##supplier.state##']     = $supplier->getField('state');
+                    $tmp['##supplier.country##']   = $supplier->getField('country');
+                    $tmp['##supplier.comments##']  = $supplier->getField('comment');
```

Comment:
Unless I am wrong, you could replace this by `$tmp = self::getActorData($supplier, CommonITILActor::ASSIGN, 'supplier');`.
It is not a problem if there are some more tags, you just have to ensure that all existing tags are kept.